### PR TITLE
Use BIT macro everywhere

### PR DESCRIPTION
**Pull request by crawfxrd**

### CODE REVIEW

File: scripts/coccinelle/bit-macro.cocci [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@@ expression val; @@
+-(1 << val)
++BIT(val)
+
+@@ expression val; @@
+-(1U << val)
++BIT(val)
+
+@@ expression val; @@
+-1 << val
```
Comment (jackpot51) [inline scripts/coccinelle/bit-macro.cocci:13]: This one seems potentially dangerous, it could over-match

Comment (crawfxrd) [reply scripts/coccinelle/bit-macro.cocci:13]: I'm not entirely sure how it matches, but it seems to match `1` exactly, as `0b11 << N` values were not modified.

**Review by jackpot51 — DISMISSED** [review]
Just need an approval from @system76/quality-assurance for this. It looks good to me

**Comment by crawfxrd** [comment]
Rebased on master

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**